src/CampaignAdmin.php

```diff
@@ -38,6 +41,8 @@ class CampaignAdmin extends LeftAndMain implements PermissionProvider
         'EditForm',
         'campaignEditForm',
         'campaignCreateForm',
+        'campaignSearchForm',
```

Comment:
We don't use `campaignSearchForm` directly - but we need it to be in `allowed_actions` so that `$this->Link('schema/campaignSearchForm')` doesn't complain at us.